pkg/cmd/cli/cmd/process.go

```diff
@@ -178,8 +178,7 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 			if len(templateName) > 0 {
 				sourceName = namespace + "/" + templateName
 			}
-			fmt.Fprintf(cmd.Out(), "unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
-			continue
+			return fmt.Errorf("unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
```

Comment:
This was due to:
 - support of lists of templates was a side-effect of some other change and not something we need to support (see above)
 - any errors in single inputs (like names of parameters) clearly means that the processed template items that would have been created are most likely not what the user wanted

Should we explicitly only look at the first item in the list?